### PR TITLE
escape hatch through update

### PR DESCRIPTION
**Pull request by nuevoalex**

### CODE REVIEW

File: source/contracts/Controller.sol [hunk 1]
```diff
@@ -127,26 +124,11 @@ contract Controller is IController {
         return true;
     }
 
-    function switchModeSoOnlyEmergencyStopsAndEscapeHatchesCanBeUsed() public devModeOwnerOnly returns (bool) {
+    function switchToCentralizedMode() public devModeOwnerOnly returns (bool) {
```
Comment (tinybike) [inline source/contracts/Controller.sol]: Should this be `switchToDecentralizedMode`?  Or, a better name might be something like, `switchOffDevMode`.

File: source/contracts/Controller.sol [hunk 2]
```diff
@@ -127,26 +124,11 @@ contract Controller is IController {
...
         whitelist[owner] = false;
```
Comment (tinybike) [inline source/contracts/Controller.sol]: Should this use `Controller.removeFromWhitelist` instead of editing `whitelist` directly?

Comment (nuevoalex) [reply source/contracts/Controller.sol]: Yea that makes sense. Will change

File: source/contracts/Controller.sol [hunk 3]
```diff
@@ -120,30 +117,15 @@ contract Controller is IController {
     function transferOwnership(address _newOwner) public onlyOwnerCaller returns (bool) {
         // if in dev mode, blacklist old owner and whitelist new owner
         if (whitelist[owner]) {
-            whitelist[owner] = false;
+            removeFromWhitelist(owner);
             whitelist[_newOwner] = true;
```
Comment (tinybike) [inline source/contracts/Controller.sol]: Suggest using `addToWhitelist(_newOwner)` instead of editing `whitelist` directly.

**Pull request merged**